SingularConfig.js

```diff
@@ -33,13 +34,19 @@ export class SingularConfig {
         this.globalProperties = {}
         this.collectOAID = false;
         this.enableLogging = false;
+        this.clipboardAttribution = false;
     }
 
     withSessionTimeoutInSec(sessionTimeout) {
         this.sessionTimeout = sessionTimeout;
         return this;
     }
 
+    withClipboardAttribution() {
```

Comment:
could it be better to give the dev the option to pass a boolean value? if they have some logic on their side to decide? the current impl is limiting IMO

Reply:
its false by default